### DISCUSSION TITLE
error MSG when add or edit modules

### DISCUSSION THREAD

**Discussion by tamermoris** [templaza/astroid-framework#772] [Q&A] [answered]
I think there is a bug with some modules. when add or edit modules shwo this msg
Astroid version 3.1.1 and 3.1.2

Best regards

![33](https://github.com/user-attachments/assets/6682ddd6-696a-4ae5-a3e9-259e5bbfa63f)

**Comment by sonvnn** [comment, marked as answer]
@tamermoris this dialog remind you click "Apply" to save your subform. If you skip it your subform will not be saved.

**Reply by tamermoris** [reply]
thank you
